packages/pds/tests/phone-verification.test.ts

```diff
@@ -181,114 +185,151 @@ describe('phone verification', () => {
   })
 
   describe('self-verification', () => {
```

Comment:
maybe a test in here for a poorly signed JWE? like encrypting/signing it with the wrong key?

Reply:
Done deal!